### DISCUSSION TITLE
realit

### DISCUSSION THREAD

**Discussion by sharklxw** [XTLS/Xray-core#2601] [General]
问好各位

**Comment by chika0801** [comment]
> 但是这样，使用谷歌搜索，有的网站打不开，关了代理又得打开了

不知道你客户端的环境是什么，应该和你客户端配置有关。

> 另外设置了"spiderX": "" 爬虫初始路径与参数，比如/wp-loginr.php,这样过墙的时候，我的流量就是在登录wordpress了？ 对吗？

这问题你去看官方REALITY最后一段话理解一下它的意思了来了。
